src/main/jbake/content/blog/2016/11/25/hawkular-apm-on-openshift.adoc

```diff
@@ -0,0 +1,82 @@
+= Using Hawkular APM on OpenShift
+Juraci Paixão Kröhling
+2016-11-25
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, microservice
+
+During the past few weeks, we've been improving how Hawkular APM can be used into OpenShift, making it easier after each iteration. At the
```

Comment:
@pilhuhn : see, I make the same mistakes in German and English :) Bonus points for consistency?